client/src/pages/Home.js

```diff
@@ -24,9 +28,12 @@ export function Home() {
 
 	return (
 		<main role="main">
+
```

Comment:
it looks like there's some issue with the indentation here, can we try to sort it out plz.